api/v1alpha3/zz_generated.conversion.go

```diff
@@ -1500,7 +1500,7 @@ func Convert_v1beta1_SpotVMOptions_To_v1alpha3_SpotVMOptions(in *v1beta1.SpotVMO
 func autoConvert_v1alpha3_SubnetSpec_To_v1beta1_SubnetSpec(in *SubnetSpec, out *v1beta1.SubnetSpec, s conversion.Scope) error {
 	// WARNING: in.Role requires manual conversion: does not exist in peer-type
 	out.ID = in.ID
-	out.Name = in.Name
+	// WARNING: in.Name requires manual conversion: does not exist in peer-type
```

Comment:
do we need to handle this in v1alpha3 conversion?

Reply:
Hmm.. I can see there are some pre existing conditions that are unhandled e.g `CIDR` will add that too.

Reply:
If there's a problem it should have been caught by fuzz tests (usually)

Reply:
Oh.. Never mind.. It just exists little nested.

Reply:
Yes. I am thinking why it didn't complain. The conversion for name does not exist and it should have complained. Lurking on to the conversion tests now.